inference-engine/src/vpu/graph_transformer/src/blob_reader.cpp

```diff
@@ -95,8 +95,11 @@ void BlobReader::parse(const std::vector<char>& blob) {
         ie::InputInfo input;
         input.setInputData(std::make_shared<ie::Data>(inputData));
 
-        _networkInputs[input.name()]    = std::make_shared<ie::InputInfo>(input);
+        if (input.name().find("@shape") == std::string::npos) {
```

Comment:
Nitpick: probably it worthies to come up with a dedicated method or something. Hardcoding logic (that name contains @shape) does not look too good

Reply:
I'm not sure I've made some final solution on this, but at least I've moved this hardcoded `@shape` to a dedicated place. Please take a look.